package-lock.json

```diff
@@ -43,6 +43,47 @@
         "xmldom": "^0.6.0"
       }
     },
+    "../maptiler-client-js": {
+      "name": "@maptiler/client",
+      "version": "1.7.0",
+      "extraneous": true,
+      "license": "BSD-3-Clause",
+      "dependencies": {
+        "@rollup/pluginutils": "^5.0.5",
+        "@types/geojson": "^7946.0.10",
+        "web-merc-projection": "^1.3.2"
+      },
+      "devDependencies": {
+        "@rollup/plugin-commonjs": "^25.0.7",
+        "@rollup/plugin-json": "^6.0.1",
+        "@rollup/plugin-node-resolve": "^15.2.3",
+        "@typescript-eslint/eslint-plugin": "^6.9.0",
+        "@typescript-eslint/parser": "^6.9.0",
+        "eslint": "^8.52.0",
+        "prettier": "^3.0.3",
+        "rollup": "^4.1.4",
+        "rollup-plugin-dts": "^6.1.0",
+        "rollup-plugin-esbuild": "^6.1.0",
+        "rollup-plugin-node-globals": "^1.4.0",
+        "rollup-plugin-string": "^3.0.0",
+        "rollup-plugin-swc": "^0.2.1",
+        "serve": "^14.1.1",
+        "sharp": "^0.32.6",
+        "typedoc": "^0.25.2",
+        "typedoc-plugin-markdown": "^3.16.0",
+        "typescript": "^5.2.2",
+        "vitest": "^0.34.6"
+      }
+    },
```

Comment:
This shouldn't be here. Maybe just merge the https://github.com/maptiler/maptiler-client-js/pull/28, release v1.7.0 and upgrade directly to that in this PR (instead of v1.6.0).

Reply:
Agreed! That's so weird, I did a local test with client 1.7 but then reinstalled the official 1.6, still it left some crumbs in package-lock...